pkgs/development/compilers/llvm/common/llvm/default.nix

```diff
@@ -112,6 +112,8 @@ stdenv.mkDerivation (finalAttrs: {
 
   patches =
     lib.optional (lib.versionOlder release_version "14") ./llvm-config-link-static.patch
+    # Fix tests nondeterministically hanging: https://github.com/llvm/llvm-project/pull/132861
+    ++ lib.optional (lib.versionAtLeast release_version "17") ./llvm-exegesis-timeout.patch
```

Comment:
Please use `getVersionFile` and attach it to `pkgs/development/compilers/llvm/common/patches.nix`.